Normalize heading case (#39)
timescale/docs: timescaledb/getting-started/compress-data.md

> # 7. Create a compression policy

Reviewer: Didn't notice before that we have two of the same headings on this page. Maybe...

```suggestion
# 7. Compression policies
```

At a high level, TimescaleDB's built-in job scheduler framework will asynchronously convert recent data from an uncompressed row-based form to a compressed columnar form across chunks of TimescaleDB hypertables.

> Let's set up a compression policy on our hypertable to see how it works.

Reviewer: ```suggestion
Let's enable compression on our hypertable and then look at two ways of compressing data: with an automatic policy or manually.
```
(I'm not beholden to this wording, but I've seen confusion over enabling compression on the table vs. actually compressing the chunks. Enabling it (`ALTER TABLE`) does not compress anything in and of itself.)

> ## Create a compression policy

Reviewer: ```suggestion
## Enable TimescaleDB compression on the hypertable
```
